.github/workflows/build.yml

```diff
@@ -22,9 +23,27 @@ jobs:
     container:
       image: ghcr.io/${{ github.repository }}-bazel-ci:latest
     steps:
-    - uses: actions/checkout@v3
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        path: ${{ env.BAZEL_WS }}/${{ env.PACKAGE_NAME }}
+    # clone public dependencies
+    - name: vcs import
+      shell: bash
+      working-directory: ${{ env.BAZEL_WS }}
+      run: vcs import src < src/${PACKAGE_NAME}/.github/dependencies.repos
```

Comment:
1. Check that we are not creating a typically colcon workspace here as we are using bazel to build. So the `src` folder might not be correct.

2. `src/${PACKAGE_NAME}/.github/dependencies.repos`. The .repos file is located at ${{ env.BAZEL_WS }}/${{ env.PACKAGE_NAME }}/.github/dependencies.repos

Reply:
Fixed!